README.md

```diff
@@ -1,130 +1,11 @@
 [![Build Status](https://travis-ci.org/spcl/daceml.svg?branch=master)](https://travis-ci.org/spcl/daceml)
 [![codecov](https://codecov.io/gh/spcl/daceml/branch/master/graph/badge.svg)](https://codecov.io/gh/spcl/daceml)
-# daceml
-Machine learning powered by data-centric parallel programming. 
+[![Documentation Status](https://readthedocs.org/projects/daceml/badge/?version=latest)](https://daceml.readthedocs.io/en/latest/?badge=latest)
 
-This project adds PyTorch and ONNX model loading support to [DaCe](https://github.com/spcl/dace), and supports ONNX operators as library nodes to DaCe stateful dataflow multigraphs.
```

Comment:
README is way too empty. You need some teasers of features and links to the appropriate doc pages.